server/app/mutations/stacks/create.rb

```diff
@@ -52,9 +52,15 @@ def validate_services
 
     def execute
       attributes = self.inputs.clone
-      %w( parent parent_name ).each { |k| attributes.delete(k.to_sym)}
+      # filter out some attributes and ensure StackRevision receives only
+      # ones that are supported to avoid Mongoid::Errors::UnknownAttribute error
+      %w( parent parent_name labels ).each { |k| attributes.delete(k.to_sym)}
```

Comment:
This by the way could be `%i( parent parent_name labels ).each { |k| attributes.delete(k) }` and avoid the `.to_sym`.

Reply:
✅ RESOLVED